client/src/pages/TaggingPage.js

```diff
@@ -1,35 +1,60 @@
-import React from "react";
+import React, { useState } from "react";
 import { Button } from "../components/Button";
 import styled from "styled-components/macro";
+import { addNewTag } from "../utils/api";
 
 const Display = styled.div`
   border: solid 1px lightgray;
 `;
+const TagForm = styled.form`
+  border: solid 1px lightgray;
+`;
+
+const TaggingPage = () => {
+  const url =
+    "https://res.cloudinary.com/tagtrace/image/upload/v1606822076/TagTrace/wmcwfmhypvc06pnjtcpg.jpg";
+  const imgNr = "001";
+  const userName = "sven";
+  const [tagName, setTagName] = useState("");
+
+  const handleTagNameChange = (event) => {
+    const tagName = event.target.value;
+    console.log("tagName:", tagName);
+    setTagName(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (!tagName) {
+      console.log("No tag name");
+    } else {
+      console.log("The fabulous tagName is:", tagName);
```

Comment:
Think about switching from console.logs to something that your users can see.

Reply:
👍